### DISCUSSION TITLE
Milestones should live at the project level

### DISCUSSION THREAD

**Discussion by tleef** [community/community#8073] [Projects and Issues]
Currently if I have a milestone that spans multiple repos, I need to duplicate the milestone in each repo.  

Thankfully when grouping issues by milestone in a project table view, it groups the issues from all of the repos together so long as the milestones in each repo are named exactly the same.

Unfortunately this means that only issues can be placed in a milestone. If you have simple tasks or "items" that relate to the milestone but not to any repo, you are stuck tracking that task separately. This is exemplified by the "Cannot add items when grouped by milestone" message that you see on tables grouped by milestone.

My proposal is allow milestones to be managed at the project level. This way issues from any repo and even simple items can be added to a milestone and tracked.

Edit: Just saw that this is already on the roadmap 🙂
https://github.com/github/roadmap/issues/276

**Comment by jwest-cox** [comment]
+1 for this. We were about to raise exactly the same suggestion. We are working around this in the same way you are too, so are pleased to see it on the road map.

**Comment by zbettenbuk** [comment]
+1 for this! Milestones should be on project and not repository level

**Comment by ghost** [comment]
Duplicate of https://github.com/orgs/community/discussions/6296 (as this has more up-votes). See also [current status](https://github.com/orgs/community/discussions/6296#discussioncomment-3147914).

**Reply by pmalek** [reply]
This returns 404. Was it closed? Removed?

**Reply by tleef** [reply]
The org name changed, here's the new link https://github.com/orgs/community/discussions/6296

**Reply by ghost** [reply]
Yes, they changed the org-name and broke older links with it 🙈 Updated it now.

**Comment by djcesar** [comment]
+1

**Comment by puncoz** [comment]
+1

**Comment by EliteGoodDev** [comment]
+1 for this. We were about to raise the same suggestion. We are working around this in the same way you are too, so I'm pleased to see it on the road map.
